python/tidy/servo_tidy/tidy.py

```diff
@@ -788,6 +833,8 @@ def get_file_list(directory, only_changed_files=False, exclude_dirs=[]):
 
 
 def scan(only_changed_files=False, progress=True):
+    # check config file for errors
+    config_errors = check_config_file(CONFIG_FILE_PATH, progress)
```

Comment:
Call the new function `parse_config` after this check (which gets the config items and overrides the default config)